docs/architecture/adr/0025-ts-deprecate-enums.md

```diff
@@ -35,15 +35,15 @@ In most cases, enums are unnecessary. A readonly (`as const`) object coupled wit
 avoids both code generation and type inconsistencies.
 
 ```ts
-const CipherType = Object.freeze({
+export const CipherType = Object.freeze({
   Login: 1,
   SecureNote: 2,
   Card: 3,
   Identity: 4,
   SshKey: 5,
 } as const);
 
-export type CipherType = _CipherType[keyof typeof CipherType];
+export type CipherType = (typeof CipherType)[keyof typeof CipherType];
```

Comment:
Worth mentioning, this is _not_ what `main` currently looks like, and probably needs to be updated to match this updated convention:

https://github.com/bitwarden/clients/blob/main/libs/common/src/vault/enums/cipher-type.ts#L1-L14